src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/NestedForDepthCheck.java

```diff
@@ -48,6 +48,30 @@
  *   &lt;property name=&quot;max&quot; value=&quot;3&quot;/&gt;
  * &lt;/module&gt;
  * </pre>
+ * <p>Violation</p>
```

Comment:
please align examples and configs better for reader to understand which config causes what violations an OK's in which examples:
- config
- example
- config
- example
- ..

also, if possible, combine OK and violation in one example